python/tvm/driver/tvmc/autotuner.py

```diff
@@ -233,6 +233,11 @@ def drive_tune(args):
     args: argparse.Namespace
         Arguments from command line parser.
     """
+    if not os.path.isfile(args.FILE):
```

Comment:
Is it possible to add a unit test for this?

Reply:
@Mousius I'm a sinner, I recognize. But, seeking redemption, I have added a test case also for `tvmc compile` which I have fixed recently.

Currently the CI is failing due to a TVMCException caught in test_tune_rpc_tracker_parsing test and since I was not able to reproduce locally I've added a fix on my guessing about the root cause. Let's see if it pleases the CI.

Thanks for the quick review!